docs/features/translation_memory.rst

```diff
@@ -47,17 +51,50 @@ If you want to setup and connect to your own TM server then the
 :setting:`AMAGAMA_URL` will allow you to point to a private TM server.
 
 
+.. _translation_memory#elasticsearch_based_tms:
+
+Elasticsearch based TMs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.7
+
+Pootle can also retrieve TM matches stored on Elasticsearch based TM servers.
+These Elasticsearch based TMs require
+`Elasticsearch <https://www.elastic.co/products/elasticsearch>`_ to be
+installed and running.
+
+.. note:: Elasticsearch depends on Java. Note that some systems may ship with
+  OpenJDK, however `elasticsearch recommends using Oracle JDK
+  <https://www.elastic.co/guide/en/elasticsearch/reference/1.6/setup-service.html#_installing_the_oracle_jdk>`_.
+
+
+Pootle supports two types of Elasticsearch based TMs:
+
+- **Local TM**: (just one, named ``local``) is populated using translations
+  stored in Pootle database and every new translation gets automatically
+  imported to it.
+
+- **External TMs**: (several) are populated from translation files specifically
+  provided by the server admins, and are not automatically updated.
+
+
+Both local and external TM settings can be adjusted in
+:setting:`POOTLE_TM_SERVER`.
+
+You may want to disable Amagama by setting :setting:`AMAGAMA_URL` to ``''`` if
+you are using Elasticsearch based TMs, though the two will operate together.
+
+
 .. _translation_memory#local_translation_memory:
 
 Elasticsearch based local TM
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+++++++++++++++++++++++++++++
 
 .. versionadded:: 2.7
 
-By default this is configured to operate if local TM is available.  To use it
-you will need to install `Elasticsearch 
-<https://www.elastic.co/products/elasticsearch>`_ and populate the initial
-translation memory using the :djadmin:`update_tmserver` command:
+By default this is configured to operate if local TM is available. To use it
```

Comment:
I'm not sure this sentence makes sense. Is *Local TM is configured by default.* what you mean?